server/src/main/java/de/pnp/manager/component/item/equipable/Weapon.java

```diff
@@ -15,11 +17,13 @@ public class Weapon extends HandheldEquipableItem implements IOffensiveItem {
     /**
      * The damage modifier of this weapon.
      */
+    @PositiveOrZero
     protected final int damage;
 
     /**
      * The dice to determine the damage.
      */
+    @NotBlank
     protected final String dice;
```

Comment:
I feel like dice should be an object at some point, so we can properly validate it, and maybe it also makes sense to have "erwartungswerte" at some point.

Reply:
Added a new ticket for this